python/pyspark/ml/feature.py

```diff
@@ -139,23 +139,26 @@ class LSHParams(Params):
                           "and decreasing it improves the running performance.",
                           typeConverter=TypeConverters.toInt)
 
-    def __init__(self):
-        super(LSHParams, self).__init__()
+    def getNumHashTables(self):
+        """
+        Gets the value of numHashTables or its default value.
+        """
+        return self.getOrDefault(self.numHashTables)
+
+
+class LSH(JavaEstimator, _LSHParams, JavaMLReadable, JavaMLWritable):
```

Comment:
I guess this should be `_LSH` as well, shouldn't it? Scala counterpart is [`private[ml]`](https://github.com/apache/spark/blob/aedf090ab7635db88f02f2c8302b56c2ee517944/mllib/src/main/scala/org/apache/spark/ml/feature/LSH.scala#L64).